Documentation/operations/system_requirements.rst

```diff
@@ -19,7 +19,7 @@ Summary
 When running Cilium using the container image ``cilium/cilium``, the host
 system must meet these requirements:
 
-- `Linux kernel`_ >= 4.19.57 (or equivalent)
+- `Linux kernel`_ >= 4.19.57 or equivalent (e.g., 4.18 on RHEL8)
```

Comment:
Unsure if we need to precise the patch version (4.18.0) here. I'm not sure we actually tested the older patch versions of RHEL8.